### ISSUE TITLE
Make details in header more accessible

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#1641]
As reported in #973:

> The first element contains Temperature and clouds,this is good. The next three elements are graphics labeled with "feels like", "wind" and "humidity". The next three elements contain the current values of them. The next three elements are texts like the labels of the graphics before: "Feels like", "wind" and "humidity". These nine elements should be only three, one for every value, and spoken as ": ", for example "Feels like: 33 °F".
